expense-frontend/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM node:20 as build
+
+WORKDIR /App-frontend
+
+COPY package*.json tsconfig.json ./
+
+RUN npm install
+
+COPY . .
+
+RUN NODE_ENV=development npm i
```

Comment:
Review again these steps.

1. First you wanna install
2. Then you wanna build
3. Finally you want to keep only important resources.